x-pack/plugins/endpoint/common/types.ts

```diff
@@ -315,23 +315,34 @@ export interface EndpointEvent {
     category: string;
     type: string;
     id: string;
+    kind: string;
```

Comment:
❔ At a glance, we have three things (`category`, `type`, and now `kind`) that are basically perfect synonyms. Can we do something to make this type more readable or useful? Either changing from `string` to the things that can actually be there, or documenting the differences?

Reply:
@bkimmel I can link you to the docs that describe the differences. We could use enums I suppose if we wanted but we'd have to update them as ecs changes. I think it'd probably go better the document the differences route.